### PR TITLE
Rock mod_muc*.erl

### PR DESCRIPTION
**Pull request by fenek**
Any comments needed? :) Enjoy reading the diff! :D

### CODE REVIEW

File: apps/ejabberd/src/mod_muc_log.erl [hunk 1]
```diff
@@ -835,8 +877,10 @@ put_room_config(F, RoomConfig, Lang, _FileFormat) ->
     {Now1, Now2, Now3} = now(),
```
Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_log.erl]: According to **Compiler**:
> Warning: erlang:now/0: Deprecated BIF. See the "Time and Time Correction in Erlang" chapter of the ERTS User's Guide for more information.

File: apps/ejabberd/src/mod_muc_log.erl [hunk 2]
```diff
@@ -848,8 +892,10 @@ put_room_occupants(F, RoomOccupants, Lang, _FileFormat) ->
...
```
Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_log.erl]: According to **Compiler**:
> Warning: erlang:now/0: Deprecated BIF. See the "Time and Time Correction in Erlang" chapter of the ERTS User's Guide for more information.

Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_log.erl]: According to **Compiler**:
> Warning: erlang:now/0: Deprecated BIF. See the "Time and Time Correction in Erlang" chapter of the ERTS User's Guide for more information.

File: apps/ejabberd/src/mod_muc_light_db_mnesia.erl [hunk 3]
```diff
@@ -243,8 +242,8 @@ get_info(RoomUS) ->
 -spec force_clear() -> ok.
 force_clear() ->
     lists:foreach(fun({RoomU, RoomS}) -> ejabberd_hooks:run(forget_room, RoomS, [RoomS, RoomU]) end,
-                  mnesia:dirty_all_keys(?ROOM_TAB)),
-    lists:foreach(fun mnesia:clear_table/1, [?ROOM_TAB, ?USER_ROOM_TAB, ?BLOCKING_TAB]).
+                  mnesia:dirty_all_keys(muc_light_room)),
+    lists:foreach(fun mnesia:clear_table/1, [muc_light_room, muc_light_user_room, muc_light_blocking]).
```
Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_light_db_mnesia.erl]: According to **Elvis**:
> Line 246 is too long:     lists:foreach(fun mnesia:clear_table/1, [muc_light_room, muc_light_user_room, muc_light_blocking])..

File: apps/ejabberd/src/mod_muc_log.erl (hunk 2 above)
Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_log.erl]: According to **Compiler**:
> Warning: erlang:now/0: Deprecated BIF. See the "Time and Time Correction in Erlang" chapter of the ERTS User's Guide for more information.

Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_log.erl]: According to **Compiler**:
> Warning: erlang:now/0: Deprecated BIF. See the "Time and Time Correction in Erlang" chapter of the ERTS User's Guide for more information.

File: apps/ejabberd/include/mod_muc_light.hrl [hunk 4]
```diff
@@ -75,13 +75,17 @@
           raw_config = [] :: raw_config()
          }).
 
+-type op_config() :: #config{}.
```
Comment (fenek) [inline apps/ejabberd/include/mod_muc_light.hrl]: When I first used it, it meant "operation" but now I'm not so convinced it's the best name. `muc_light_config()` is soooooo long. :D

Comment (michalwski) [inline apps/ejabberd/include/mod_muc_light.hrl]: It's not that long :) IMO, the prettiest would be to move this type to `mod_muc_light` and name it `config`. Then you will use type like `mod_muc_light:config()`. I know this is even longer :)

Comment (fenek) [inline apps/ejabberd/include/mod_muc_light.hrl]: Why not, I'll give it a try.

File: apps/ejabberd/src/mod_muc_log.erl (hunk 2 above)
Comment (gadgetci) [inline apps/ejabberd/src/mod_muc_log.erl]: According to **Compiler**:
> Warning: erlang:now/0: Deprecated BIF. See the "Time and Time Correction in Erlang" chapter of the ERTS User's Guide for more information.

**Comment by michalwski** [comment]
Thanks!

**Pull request merged**